src/queue/eventhub/utils.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable sort-keys */
+import {
```

Comment:
Can you open a separate PR in fp-ts-kafka-js package in order to add consumer capabilities?